### ISSUE TITLE
io.socket.post header 

### ISSUE THREAD

**Issue by Naranbat1123** [balderdashy/sails#2373]
How to set io.socket.post header

**Comment by CWyrtzen** [comment]
Hi @Naranbat1123 did you figure out how to set io.socket.post header?

**Comment by sailsbot** [comment]
Thanks for posting, @Naranbat1123.  I'm a repo bot-- nice to meet you!

It has been 60 days since there have been any updates or new comments on this page.  If this issue has been resolved, feel free to disregard the rest of this message.  On the other hand, if you are still waiting on a patch, please:

  + review our [contribution guide](https://github.com/balderdashy/sails/blob/master/CONTRIBUTING.md) to make sure this submission meets our criteria (only _verified bugs_ with documented features, please;  no questions, commentary, or bug reports about undocumented features or unofficial plugins)
  + create a new issue with the latest information, including updated version details with error messages, failing tests, and a link back to [the original issue]().  This allows GitHub to automatically create a back-reference for future visitors arriving from search engines.

Thanks so much for your help!
